qa/test-db-instance-migration/test-fixture-710/pom.xml

```diff
@@ -13,6 +13,8 @@
 
   <properties>
     <camunda.version.previous>7.9.0</camunda.version.previous>
+    
+    <version.h2>1.4.190</version.h2>
```

Comment:
Should we introduce some property like `version.h2-v1` that we can reference in all older test fixtures? And should we update to 1.4.200 at least, so we're on the latest 1.4.x version?

Reply:
> Should we introduce some property like version.h2-v1 that we can reference in all older test fixtures?

Sure, that makes sense.

> And should we update to 1.4.200 at least, so we're on the latest 1.4.x version?

I don't think this is worth it. The older engines ship and are tested with 1.4.190 and for the purpose of instance migration, the version shouldn't make a difference. Lastly, 1.4.200 has already some breaking changes, e.g. MVCC flag removed.

Reply:
Makes sense, I expected something like that to be the case, actually. Thanks for the details 👍